drafts/use-cases/1.entry-point.md

```diff
@@ -20,24 +20,23 @@ in a configuration or in the input field.
 Once provided, the application is expected to connect to the entry-point of the Web API and download initial
 details on what else can be done with that API from that very point.
 
-```
+```http
 GET /api
 
 HTTP 200 OK
 ```
-```javascript
+```json
 {
     "@id": "/api",
```

Comment:
Sorry, didn't notice it before. This example should probably be changed to `GET /events` or `GET /api/events`